docs/octopus-rest-api/octopus.server.exe-command-line/configure.md

```diff
@@ -196,7 +203,7 @@ Where [<options>] is any of:
                              Set whether GitHub issue tracker integration is
                                enabled.
       --GitHubBaseUrl=VALUE  Set the base url for the Git repositories.
-      --jiraIsEnabled=VALUE  Set whether Jira Integration is
+      --jiraIsEnabled=VALUE  Set whether Jira issue tracker integration is
```

Comment:
Ahh, looks like @Silcoish made this [change manually](https://github.com/OctopusDeploy/docs/commit/0309a554726d5f75bd8732fb7fcae292c3fd90f6)...
What version is this available to public, @Silcoish?

Reply:
2020.2.0